apps/betterangels-backend/accounts/migrations/0030_alter_clientcontact_phone_number.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.0.7 on 2024-08-06 00:29
```

Comment:
these migrations can be collapsed into one. easiest way to do that is to delete them both and rerun `makemigrations`

Reply:
Although, I was able to make a single migration, it kept giving me this error when I tried to migrate it:

```
> poetry run python manage.py migrate

Operations to perform:
  Apply all migrations: account, accounts, admin, auth, common, contenttypes, guardian, notes, organizations, pghistory, post_office, sessions, shelters, sites, socialaccount, waffle
Running migrations:
  Applying accounts.0029_alter_clientcontact_phone_number_and_more...Traceback (most recent call last):
  File "/workspace/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 103, in _execute
    return self.cursor.execute(sql)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.venv/lib/python3.12/site-packages/psycopg/cursor.py", line 97, in execute
    raise ex.with_traceback(None)
psycopg.errors.InvalidTextRepresentation: invalid input syntax for type json
DETAIL:  Token "+" is invalid.
CONTEXT:  JSON data, line 1: +...

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/workspace/apps/betterangels-backend/manage.py", line 30, in <module>
    main()
  File "/workspace/apps/betterangels-backend/manage.py", line 26, in main
    execute_from_command_line(sys.argv)
```

Only when I separated the migrations, did it start working